### PR TITLE
Addresses BACKLOG-37996: WCAG main page zoom

### PR DESCRIPTION
**Pull request by mbrasil**
@pentaho/wcag  and @pentaho/millenniumfalcon  please review

Main PR: https://github.com/pentaho/pentaho-platform-plugin-interactive-reporting/pull/1015

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1735/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1735/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: impl/client/src/main/javascript/web/dojo/pentaho/common/MessageBox.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
      {
         buttons: ['btn1','btn2','btn3'],
         messageType: null,                         // options are null, ERROR, WARN, INFO
+        responsive: true, // Not intended to be set to false.
+        responsiveClasses: "dw-sm",
```
Comment (afrjorge) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/MessageBox.js:25]: license header

Comment (mbrasil) [reply impl/client/src/main/javascript/web/dojo/pentaho/common/MessageBox.js:25]: Just added the LICENSE.txt file. Thank you :)

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1735/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 8m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1735/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 7s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **73**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/common-ui/job/PR-1735/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by davidmsantos90 — APPROVED** [review]

**Pull request merged**